### PR TITLE
Add controls locking

### LINKED ISSUE #105: Option to disable Changing Inverter Charger modes.
It may be  good to have the option of disabling the changing of the Inverter Charger mode via the HTML5 app. This may prevent mistakenly changing the inverter mode.

### PR DESCRIPTION
**Pull request by margotr**
Closes #105 

Todo: 

- [x] Implement lock button (mobile & full view)
- [x] Test LockButton
- [x] Test SelectorButton
- [x] Lock icons as image/svg instead of import 
- [x] Lock icon stroke width

Mobile view (Lock button in footer): 
Screenshot 2020-09-23 at 15 16 23

Desktop view (Lock button in header):
Screenshot 2020-09-23 at 15 17 38

### CODE REVIEW

**Comment by margotr** [comment]
Update: 
[ ] Fix bug where disabled state does not actually prevent clicking the button...

File: src/app/components/SelectorButton/SelectorButton.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+const getSelectorButtonWithLockedContext = (context = {}) => {
+  jest.doMock("../../contexts", () => {
+    return {
+      LockContext: {
+        Consumer: props => props.children(context)
+      }
+    }
+  })
+  require("./SelectorButton").default
```
Comment (AlexImb) [inline src/app/components/SelectorButton/SelectorButton.test.js:17]:
```suggestion

```

Why is this `require` required?

Comment (margotr) [reply src/app/components/SelectorButton/SelectorButton.test.js:17]: Essentially it re-imports the selector button with context applied. This is hacky context/jest stuff, but it's the solution used by most people right now.

**Review by AlexImb — APPROVED** [review]

**Comment by AlexImb** [comment]
Rebased externally onto `develop`

**Pull request closed without merging**